src/Controller/Api/CacheController.php

```diff
@@ -171,10 +168,17 @@ private function getPurgeUrlsForMajorVersion(float $version): array
     }
 
     /**
-     * @param string $route
-     * @param array $parameters
-     * @return string
+     * Deletes the releases.json in the cache
      */
+    private function deleteReleases(): void
+    {
+        $filesystemAdapter = new \Symfony\Component\Cache\Adapter\FilesystemAdapter();
```

Comment:
Why the FQCN here instead of a `use` above?

Reply:
I'm also curious why Rector things the method needs to be moved.  It doesn't seem to do anything.

Reply:
Feeling the same about the FQCN, I also prefer having an use statement instead!

Not sure about moving that method, didn't see the reason here. Ordering private, protected and public methods makes sense. But well, it doesn't hurt...

Reply:
changed